nbconvert/exporters/templateexporter.py

```diff
@@ -107,7 +107,7 @@ def _template_file_default(self):
 
     default_template = Unicode(u'').tag(affects_template=True)
 
-    template_path = List(['.']).tag(config=True, affects_environment=True)
+    template_path = List(['.', '/']).tag(config=True, affects_environment=True)
```

Comment:
I'd like to find a way to handle absolute paths that's not also searching relative paths starting from the root dir. Also, we probably need to think about Windows.